opsdroid/database/mongo/tests/test_mongo.py

```diff
@@ -0,0 +1,77 @@
+import pytest
+
+from opsdroid.database.mockmodules.mongo.mongo_database import DatabaseMongoTest
+from opsdroid.database.mongo import DatabaseMongo
+
+
+@pytest.fixture()
+def database(config):
+    return DatabaseMongo(config)
+
+
+@pytest.mark.parametrize("config", [{"example_item": "test"}])
+def test_init(database):
+    """Test that the database is initialised properly."""
+    assert database.name == "mongo"
+    assert database.config["example_item"] == "test"
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize("config", [{}])
+async def test_connect(database):
+    """Test that the mongo database has implemented connect function properly"""
+    try:
+        await database.connect()
+    except NotImplementedError:
+        raise Exception
+    else:
+        pass
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize("config", [{}])
+async def test_get(database):
+    database.database = {}
+    database.database["test"] = DatabaseMongoTest({})
```

Comment:
```suggestion
    database.database["mongo"] = DatabaseMongoTest({})
```

I just think it would make more sense to name it mongo than "test", what do you think?

Reply:
Hi, @FabioRosado , thanks for the reply. The name "test" was in the unittest counterpart, so I kept it same. Happy to change it since it makes more sense.

Reply:
Oh I understand, I haven't look at the old tests haha would be good to change it to mongo then 😁